iac/terraform/aws/acm.tf

```diff
@@ -0,0 +1,11 @@
+data "aws_acm_certificate" "certificate" {
+  domain      = var.bloodconnect_domain
+  statuses    = ["ISSUED"]
+  most_recent = true
+  provider = aws.virginia
+}
+
+provider "aws" {
```

Comment:
This is not allowed in child module block. That's the reason of security error.

Reply:
Done changing